src/uk/gov/hmcts/cmc/integrationtests/IntegrationTests.groovy

```diff
@@ -119,9 +117,19 @@ class IntegrationTests implements Serializable {
             """
   }
 
+  private void archiveDockerStatus() {
+    steps.sh "mkdir -p output && ${dockerComposeCommand()} ps > output/docker-status.txt"
+    steps.archiveArtifacts 'output/docker-status.txt'
+  }
+
   private void archiveDockerLogs() {
-    steps.sh "mkdir -p output && ${dockerComposeCommand()} logs --no-color > output/logs.txt"
-    steps.archiveArtifacts 'output/logs.txt'
+    steps.sh """
+             mkdir -p output
+             ${dockerComposeCommand()} logs --no-color > output/docker-logs.txt
```

Comment:
Does it give us any value to have all logs in one file if we're splitting them anyway later?

Reply:
Not that much value, at least from my experience, decided to keep it in case someone is chasing timing of the events.